### ISSUE TITLE
Use a LLM different from OpenAI

### ISSUE THREAD

**Issue by NeuroinformaticaFBF** [TsinghuaC3I/LLM4BioHypoGen#1]
Hello there!

I was trying to use your code to run some test.
Deprecations aside, I see that it is all coded to run with OpenAI models.

Is there a quick way to switch to other models, like for example Llama?

**Comment by iseesaw** [comment]
Yes, you can run the Llama model using [vLLM on an OpenAI-compatible server](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html). This will allow you to call the API just as our code does, enabling compatibility with Llama.
